code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -123,6 +123,24 @@
 	results = list(/datum/reagent/nitrous_oxide = 5)
 	required_reagents = list(/datum/reagent/ammonia = 2, /datum/reagent/nitrogen = 1, /datum/reagent/oxygen = 2)
 	required_temp = 525
+	optimal_temp = 550
+	overheat_temp = 575
+	temp_exponent_factor = 0.2
+	purity_min = 0.3
+	thermic_constant = 35 //gives a bonus 15C wiggle room
+	rate_up_lim = 25 //Give a chance to pull back
+	
+/datum/chemical_reaction/nitrous_oxide/overly_impure(datum/reagents/holder, datum/equilibrium/equilibrium)
+	. = ..()
+	var/turf/exposed_turf = get_turf(holder.my_atom)
+	if(!exposed_turf)
+		return
+	exposed_turf.atmos_spawn_air("n2o=[equilibrium.step_target_vol/2];TEMP=[holder.chem_temp]")
+	clear_products(holder, equilibrium.step_target_vol)
+
+/datum/chemical_reaction/nitrous_oxide/overheated(datum/reagents/holder, datum/equilibrium/equilibrium)
+	return //When I refactor explosions, this will be used, but for now, the alternative reaction will catch the high temperature, so we don't need to do anything. This is so that on lookup, the overheat value will be displayed.
```

Comment:
This comment seems more like a note to yourself than something that's useful for other developers.

Reply:
It's more to explain the pointless overheat proc, but I'm happy with removing it.

Reply:
Can you find a way to explain it in a way that doesn't mention future refactors? As in, document why this proc is empty *now*.

Reply:
Will do!